contrib/runners/action_chain_runner/setup.py

```diff
@@ -42,5 +42,13 @@
     zip_safe=False,
     include_package_data=True,
     py_modules=['action_chain_runner'],
-    scripts=[]
+    data_files=[
+        ('metadata', ['runner.yaml'])
+    ],
+    scripts=[],
+    entry_points={
+        'st2common.runners.runner': [
+            'action_chain_runner = action_chain_runner',
```

Comment:
maybe it makes sense to set entrypoint name to runner name rather than a runner module name? Or better yet, merge runner name and runner module name.

The problem I see with current approach is that there's no way to figure out which runner do you need until you install and enumerate all of them. Then you left with a bunch of runners that you either bundle with lambda function bloating it or have to uninstall \ exclude in some other way.

It's mot a blocker at the moment, sure, but it's something that is nagging me and I don't see us being done with runners just yet.